packages/graphql/tests/integration/deprecated/fulltext.int.test.ts

```diff
@@ -0,0 +1,745 @@
+                driver,
+                driverConfig: { database: databaseName },
+                options: { create: true },
+            })
+        ).resolves.not.toThrow();
+
+        // Previously this would have thrown, but the operation should be idempotent
+        await expect(
+            neoSchema.assertIndexesAndConstraints({
+                driver,
+                driverConfig: { database: databaseName },
+                options: { create: true },
+            })
+        ).resolves.not.toThrow();
+
+        const session = driver.session({ database: databaseName });
+
+        const cypher = `
+            SHOW INDEXES yield
+                name AS name,
+                type AS type,
+                entityType AS entityType,
+                labelsOrTypes AS labelsOrTypes,
+                properties AS properties
+            WHERE name = "${indexName}"
+            RETURN {
+                 name: name,
+                 type: type,
+                 entityType: entityType,
+                 labelsOrTypes: labelsOrTypes,
+                 properties: properties
+            } as result
+        `;
+
+        try {
+            const result = await session.run(cypher);
+
+            const record = result.records[0].get("result") as {
+                name: string;
+                type: string;
+                entityType: string;
+                labelsOrTypes: string[];
+                properties: string[];
+            };
+
+            expect(record.name).toEqual(indexName);
+            expect(record.type).toBe("FULLTEXT");
+            expect(record.entityType).toBe("NODE");
+            expect(record.labelsOrTypes).toEqual([type.name]);
+            expect(record.properties).toEqual(["title"]);
+
+            await session.run(`
+                CREATE (:${type.name} { title: "${title}" })
+            `);
+        } finally {
+            await session.close();
+        }
+    });
+
+    test("should throw when index is missing fields when used with create option", async () => {
+        // Skip if multi-db not supported
+        if (!MULTIDB_SUPPORT) {
+            console.log("MULTIDB_SUPPORT NOT AVAILABLE - SKIPPING");
+            return;
+        }
+
+        const indexName = generate({ readable: true, charset: "alphabetic" });
+        const type = generateUniqueType("Movie");
+
+        const typeDefs = gql`
+            type ${type.name} @fulltext(indexes: [{ name: "${indexName}", fields: ["title", "description"] }]) {
+                title: String!
+                description: String!
+            }
+        `;
+
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+        await neoSchema.getSchema();
+
+        const session = driver.session({ database: databaseName });
+
+        try {
+            await session.run(
+                [`CREATE FULLTEXT INDEX ${indexName}`, `IF NOT EXISTS FOR (n:${type.name})`, `ON EACH [n.title]`].join(
+                    " "
+                )
```

Comment:
Is there a reason for having this as an array with join?

Reply:
I don't know the reason for this. These are the old full-text tests and I just threw them in a deprecated folder to be deleted when the deprecated version is removed in 4.0.

As they're going to be deleted soon anyway I don't think it's worth changing this